src/sql/workbench/contrib/dashboard/browser/core/dashboardWidget.ts

```diff
@@ -42,6 +42,7 @@ export interface TabConfig extends IDashboardTab {
 	canClose: boolean;
 	actions?: Array<Action>;
 	iconClass?: string;
+	type?: 'tab' | 'header';
```

Comment:
Would suggest defining an actual type for this since you're already using it in multiple places.